<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_widget</class>
 <widget class="QWidget" name="search_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <item>
      <layout class="QHBoxLayout" name="search_layout" stretch="1,9,1,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>English Word</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_line"/>
       </item>
       <item>
        <widget class="QPushButton" name="search_button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="result_layout" stretch="7,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,2">
           <item>
            <widget class="QLabel" name="word_label">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Word</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="pronounciation_layout" stretch="9,1">
             <item>
              <widget class="QLabel" name="pronounciation_label">
               <property name="text">
                <string>IPA pronounciation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sound_button">
               <property name="text">
                <string>Sound</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="main_layout">
           <item>
            <layout class="QVBoxLayout" name="sense_1">
             <item>
              <widget class="QLabel" name="definition_label_1">
               <property name="text">
                <string>Definition: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sentence_label_1">
               <property name="text">
                <string>Sentence: </string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>My Example</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="my_example_1"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="sense_2">
             <item>
              <widget class="QLabel" name="definition_label_2">
               <property name="text">
                <string>Definition: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sentence_label_2">
               <property name="text">
                <string>Sentence: </string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>My Example</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="my_example_2"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="sense_3">
             <item>
              <widget class="QLabel" name="definition_label_3">
               <property name="text">
                <string>Definition: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sentence_label_3">
               <property name="text">
                <string>Sentence: </string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>My Example</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="my_example_3"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,1">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="match_results"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="variants_result"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
